### PR TITLE
Completed: New onboarding screen

### PR DESCRIPTION
**Pull request by iosdevashok**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-TODO)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/TODO)

### Description
TODO

### Pull requests checks where applicable
- [ ] Fill in the three TODOs above (tickets number and description of your work)
- [ ] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by iosdevashok** [comment]
https://user-images.githubusercontent.com/102407838/227279053-91b3e554-b11e-4429-970b-cc62b3fbfdbc.mp4

File: Client/Frontend/Intro/IntroViewController.swift
Comment (madtocc) [inline Client/Frontend/Intro/IntroViewController.swift:1]: Whenever you remove code, avoid commenting out the code. You can erase the code not used. If we want to throw back we can revert.

Comment (madtocc) [inline Client/Frontend/Intro/IntroViewController.swift:1]: Can we still use the page control? Looks like we have lost the functionality to scroll and move to the other page.

File: Client/Frontend/Intro/OnboardingViewController.swift
Comment (madtocc) [inline Client/Frontend/Intro/OnboardingViewController.swift:1]: the last page should have a "Done" or "Next" button at the bottom to be able to dismiss the on boarding modal.

**Review by madtocc — CHANGES_REQUESTED** [review]

**Review by madtocc — CHANGES_REQUESTED** [review]
All good now, however when scrolling manually instead of clicking next the page indicator does not change.
Scroll back and forward should update the current page index.
currently just hitting next updates the page indicator.

**Comment by madtocc** [comment]
We should keep the manual scroll, the scroll should update the indicator...

**Comment by madtocc** [comment]
@iosdevashok 
There are 2 page indicators. We don't need the one on the bottom (grey).

Expected:
When clicking "Next" it should move forward and change the orange indicator.
When you swipe left or right the orange indicator should change according to the page.

Current:
When you click on next it changes the orange indicator ✅
manually swiping it does not change the orange indicator ❌ 
Shows a grey indicator next to the bottom of the page  ❌

https://user-images.githubusercontent.com/12486417/228969210-20b0ab83-25a1-41dc-a4ea-e6f439c33635.mov

**Pull request merged**